tests/pytests/test_hayabusa2_drivers.py

```diff
@@ -8,177 +8,26 @@
-                                   'AngularVelocity' : [[-3.3566444941696236e-05, 3.30201152018665e-05, 2.987092640139532e-06]]},
-            'BodyRotation': {'TimeDependentFrames': [10013, 1],
-                             'CkTableStartTime': 502404366.34876,
-                             'CkTableEndTime': 502404366.34876,
-                             'CkTableOriginalSize': 1,
-                             'EphemerisTimes': [502404366.34876287],
-                             'Quaternions': [[-0.1979938370409085, -0.0007582477662989689, 0.00015456662316054356, -0.9802029594238645]],
-                             'AngularVelocity' : [[1.128586084254313e-07, -1.9793051740218293e-10, 7.292106285199177e-05]]},
-            'InstrumentPosition': {'SpkTableStartTime': 502404366.34876,
-                                   'SpkTableEndTime': 502404366.34876,
-                                   'SpkTableOriginalSize': 1,
-                                   'EphemerisTimes': [502404366.34876287],
-                                   'Positions': [[30.15558294075686, -21415.864299596455, 29138.041066520385]],
-                                   'Velocities': [[1.5409457644671734, 5.3312261011154565, -3.5389092329306164]]},
-            'SunPosition': {'SpkTableStartTime': 502404366.34876,
-                            'SpkTableEndTime': 502404366.34876,
-                            'SpkTableOriginalSize': 1,
-                            'EphemerisTimes': [502404366.34876287],
-                            'Positions': [[-48916727.046504386, -127647230.31032252, -55336271.675298646]],
-                            'Velocities': [[28.58340952406857, -8.972886737522412, -3.888902421092297]]}
-        }
-    }
-}
-
+from conftest import get_image_kernels, convert_kernels, get_image_label, compare_dicts, get_isd
 
 @pytest.fixture(scope='module')
 def test_kernels():
-    updated_kernels = {}
-    binary_kernels = {}
-    for image in image_dict.keys():
-        kernels = get_image_kernels(image)
-        updated_kernels[image], binary_kernels[image] = convert_kernels(kernels)
-        yield updated_kernels
-    for kern_list in binary_kernels.values():
-        for kern in kern_list:
-            os.remove(kern)
+    kernels = get_image_kernels('hyb2_onc_20151203_084458_w2f_l2a')
+    updated_kernels, binary_kernels = convert_kernels(kernels)
+    yield updated_kernels
+    for kern in binary_kernels:
+        os.remove(kern)
 
 
-@pytest.mark.xfail
 @pytest.mark.parametrize("label_type", ['isis3'])
-@pytest.mark.parametrize("formatter", ['isis'])
-@pytest.mark.parametrize("image", image_dict.keys())
-def test_hayabusa_load(test_kernels, label_type, formatter, image):
-    label_file = get_image_label(image, label_type)
+def test_hayabusa_load(test_kernels, label_type):
+    label_file = get_image_label('hyb2_onc_20151203_084458_w2f_l2a', label_type)
+    usgscsm_isd_str = ale.loads(label_file, props={'kernels': test_kernels}, verbose=False)
```

Comment:
Super pedantic, but since the usgscsm was an old format type and it's now generating an "ale" isd string, I think it would be less confusing to call this variable isd_str (or something else without usgscsm in it.)